lib/cmd/fh3/connections/create.js

```diff
@@ -0,0 +1,78 @@
+/* globals i18n */
+var common = require("../../../common");
+var fhreq = require("../../../utils/request");
+
+module.exports = {
+  'desc' : i18n._('Create connections'),
+  'examples' :
+  [{
+    cmd : 'fhc connections create --client=<client> --cloud=<cloud> --type=<type> --env=<env> --project=<project> --connection=<connection>',
+    desc : i18n._('Create the <connection> for the <client> with the <type> and the <cloud> from the <project> deployed on the <env>')
+  },{
+    cmd : 'fhc connections create --client=<client> --cloud=<cloud> --type=<type> --env=<env> --project=<project>',
+    desc : i18n._('Create a a new sequencially connection for the <client> with the <type> and the <cloud> from the <project> deployed on the <env>')
+  }
+  ],
+  'demand' : ['client','cloud','type','env','project'],
+  'alias' : {
+    'client' : 'cli',
+    'cloud' : 'clo',
+    'type' : 't',
+    'env' : 'e',
+    'project' : 'p',
+    'connection' : 'c',
+    'json': 'j',
+    0 : 'client',
+    1 : 'cloud',
+    2 : 'type',
+    3 : 'env',
+    4 : 'project',
+    5 : 'connection'
+  },
+  'describe' : {
+    'client' : i18n._('Unique 24 character GUID of the project'),
+    'cloud' : i18n._('Unique 24 character GUID of the connection'),
+    'type' : i18n._('"Client app type : [android, ios, cordova]"'),
+    'env' : i18n._('Unique 24 character GUID of the cloud application'),
+    'project' : i18n._('Unique 24 character GUID of the cloud application'),
+    'connection' : i18n._('Connection Tags must be in Semantic Version format, e.g. 0.0.1. See: http://semver.org'),
+    'json' : i18n._("Output in json format")
+  },
+  'customCmd' : function(params,cb) {
+    var url = '/box/api/connections/' ;
+    params.type = params.type.toLowerCase();
+    if ( params.type !== 'android' && params.type !== 'ios' && params.type !== 'cordova' ) {
```

Comment:
I don't think there is such a concept as "cordova" for creating connections?

In the platform if you create an iOS application using Cordova the connection tag `destination` is "`ios` not `html5`.

Reply:
In other words I suggest removing `cordova` option and just using `["html5", "ios", "android"]` since this seems to be what the platform does?

Reply:
@evanshortiss in all commands in FHC the platforms are defined as android, ios, and Cordova.
IMHO, the users/customers will not able to know what means html5. When the user chooses Cordova, the value informed is html5. Wdyt? Is it makes sense for you?

Reply:
👍 I think you are correct. The idea was a change for the user only to use the Cordova instead o html5 to make it more intuitive. However, I think is better to keep the html5 as you suggested here.